en/test_and_ci/test_flights.md

```diff
@@ -28,30 +28,32 @@ The tests performed for each platform are linked below:
-* 3DR Iris+
+Multicopter
+
+Frame | Flight Controller | UUID
+--- | --- | ---
+[DJI F450](https://www.getfpv.com/dji-flamewheel-f450-basic-kit.html) | [Pixhawk 4 Mini](https://docs.px4.io/en/flight_controller/pixhawk4_mini.html) | 002400283335510A33373538 (f450-v3)
+[DJI F450](https://www.getfpv.com/dji-flamewheel-f450-basic-kit.html) | [Pixhawk 1](https://docs.px4.io/en/flight_controller/pixhawk.html) | 000100000000363533353336510900500021 (f450-v3)
+[DJI F450](https://www.getfpv.com/dji-flamewheel-f450-basic-kit.html) | [Cube](https://docs.px4.io/en/flight_controller/pixhawk-2.html) (Pixhawk 2.1) | 00010000000033343537313751050040001c (F450 Pixhawk v2 cube)
+[DJI F450](https://www.getfpv.com/dji-flamewheel-f450-basic-kit.html) | [Pixracer](https://docs.px4.io/en/flight_controller/pixracer.html) | 00010000000037373430333551170037002a (F450-Pixracer)
+[DJI F450](https://www.getfpv.com/dji-flamewheel-f450-basic-kit.html) | [Pixhawk 3 Pro](https://docs.px4.io/en/flight_controller/pixhawk3_pro.html) | 000100000000303236353136510500180036 (Pixhawk pro)
+[DJI F450](https://www.getfpv.com/dji-flamewheel-f450-basic-kit.html) | [Pixhack V3](https://docs.px4.io/en/flight_controller/pixhack_v3.html) | 003200293036511638363834 (f450-v5-m)
+[DJI F450](https://www.getfpv.com/dji-flamewheel-f450-basic-kit.html) | [Pixhawk 4](https://docs.px4.io/en/flight_controller/pixhawk4.html) | 000200000000383339333038510700320016 (F450-v5)
+[DJI F450](https://www.getfpv.com/dji-flamewheel-f450-basic-kit.html) | [Pixhawk 4 Mini](https://docs.px4.io/en/flight_controller/pixhawk4_mini.html) | 0002000000003432333830385115003a0033 (F450-v5-m)
+[DJI F450](https://www.getfpv.com/dji-flamewheel-f450-basic-kit.html) [UAVCAN](https://zubax.com/technologies/uavcan) | [Pixhawk 4](https://docs.px4.io/en/flight_controller/pixhawk4.html) | 000200000000323634353237511800200021 (F450-Pixhawk4)
+HoliBro [QAV250](https://docs.px4.io/en/frames_multicopter/holybro_qav250_pixhawk4_mini.html) | [Pixhawk 4 Mini](https://docs.px4.io/en/flight_controller/pixhawk4_mini.html) |000200000000343233383038511500420032 (f450-v5-m)
+NXP Semiconductor [KIT-HGDRONEK66](https://www.nxp.com/applications/solutions/industrial/unmanned-aerial-vehicles-uavs/uavs-drones-and-rovers/rddrone-fmuk66-px4-robotic-drone-fmu-reference-design:RDDRONE-FMUK66) ("[Hovergames](https://www.hovergames.com/)")| [RDDRONE-FMUK66](https://www.nxp.com/products/processors-and-microcontrollers/arm-based-processors-and-mcus/kinetis-cortex-m-mcus/k-seriesperformancem4/k6x-ethernet/rddrone-fmuk66-px4-robotic-drone-fmu-reference-design:RDDRONE-FMUK66?tid=vanRDDRONE-FMUK66) | 00030016ffffffffffff4e45362050130029
+
+Fixed Wing
+
+Frame | Flight Controller | UUID
+--- | --- | ---
+[Phantom Wing](https://hobbyking.com/en_us/phantom-fpv-flying-wing-epo-airplane-1550mm-v2-kit.html) | [Pixhawk 1](https://docs.px4.io/en/flight_controller/pixhawk.html) | 0001000000003035333330365104003c0020 (f450-v2)
```

Comment:
@Tony3dr The UUID of (f450-v2) is odd given this is a wing, but assuming it is accurate, I am going to merge this PR - which is otherwise useful. 

It should be up to date based on your information - thanks. Can you please give it a final sanity check though?